src/docs/openshift-projects-and-access/common-platform-requests.md

```diff
@@ -0,0 +1,59 @@
+
+There’s a fee for each user that has access to a private repository in the bcgov-c organization, only grant access to users that need it. The fee for all users is currently covered by the OCIO.
+
+There are a limited number of spots available in the bcgov-c organization. If there are no spots remaining, you’ll receive an error message when you try to add a new user. If this occurs, send a request to [PlatformServicesTeam@gov.bc.ca](mailto:platformservicesteam@gov.bc.ca) for an additional spot to the Platform Services Team. Once we confirm your request has been completed, you can add the new user to your repository.
+
+The Platform Services team removes bcgov-c organization access for users that have not been active for 6 months. Once access has been removed, users must be re-invited to the organization and re-added to a repository to gain access.
+
+## Request a new private repository in the bcgov-c GitHub organization
+
+Only the Platform Services team can create new private repositories in the bcgov-c GitHub organization.
+
+[Submit a request to create a private repository.](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FBCDevOps%2Fdevops-requests%2Fissues%2Fnew%3Fassignees%3Dcaggles%252C%2BShellyXueHan%252C%2Bmitovskaol%252C%2Bpatricksimonian%26labels%3Dgithub-repo%252C%2Bpending%26template%3Dgithub_repo_request.md%26title%3D)
+
+## Request Keycloak Single-Sign-On (SSO) integration with your application
+
+Get access to the Keycloak SSO login service that connects your users to your applications using government credentials, including IDIR.
+
+[Submit a request for Keycloak single-sign-on integration](https://github.com/bcgov/sso-keycloak/wiki/SSO-Onboarding) with your application.
+
+## Request a new project on OpenShift or make changes to an existing project
+
+Request a new project on the B.C. Government Private Cloud PaaS, make changes to an existing project in OpenShift, request a resource quota increase for your app or update information for an existing project.
+
+Submit a request for a new project or make changes to an existing project on OpenShift through the Platform Product Registry. 
+
+Follow the steps involved to [request a quota increase for an OpenShift project set](../automation-and-resiliency/request-quota-increase-for-openshift-project-set.md).
+
+## Request to remove OpenShift or B.C. Government GitHub organization membership
+
+Remove a team member’s access to OpenShift or a B.C. Government GitHub repository.
+
+[Submit a request to remove GitHub or OpenShift membership.](https://github.com/BCDevOps/devops-requests/issues/new?assignees=caggles%2C+ShellyXueHan%2C+mitovskaol%2C+patricksimonian&labels=github-membership%2Copenshift-access&template=github_user_access_request.md&title=)
```

Comment:
```suggestion
[Request to remove OpenShift Access and or BCGOV GitHub Organization Membership](https://github.com/BCDevOps/devops-requests/issues/new/choose)
```

Reply:
As above, linking to the general issue creation page rather than the markdown file for the issue. Matching the link name to the issue name.